lib/explorer/series.ex

```diff
@@ -1274,18 +1266,18 @@ defmodule Explorer.Series do
       iex> Explorer.Series.rolling_sum(s, 4)
       #Explorer.Series<
         integer[10]
-        [nil, nil, nil, 10, 14, 18, 22, 26, 30, 34]
+        [1, 3, 6, 10, 14, 18, 22, 26, 30, 34]
       >
 
       iex> s = 1..10 |> Enum.to_list() |> Explorer.Series.from_list()
       iex> Explorer.Series.rolling_sum(s, 2, [1.0, 2.0])
       #Explorer.Series<
-        integer[10]
-        [nil, 5, 7, 11, 13, 17, 19, 23, 25, 29]
+        float[10]
+        [1.0, 5.0, 8.0, 11.0, 14.0, 17.0, 20.0, 23.0, 26.0, 29.0]
       >
   """
-  def rolling_sum(series, window_size, weights \\ nil, ignore_nil? \\ true),
-    do: apply_impl(series, :rolling_sum, [window_size, weights, ignore_nil?])
+  def rolling_sum(series, window_size, weights \\ nil, min_periods \\ 1, center \\ false),
```

Comment:
I am wondering if we should change the API to keyword lists. Does dplyr have a similar API? If so, how does it look?

Reply:
I agree with that :+1: 
But maybe we should do in another PR :thinking: 

And dplyr doesn't seems to have rolling aggregates: https://dplyr.tidyverse.org/articles/window-functions.html?q=rolling#types-of-window-functions
But in general function options looks like keyword lists.

Reply:
Agreed it should probably be in another PR and also agreed that this should be keyword lists. And yeah, no rolling aggregates in dplyr -- they suggest using [RcppRoll](https://cran.r-project.org/web/packages/RcppRoll/RcppRoll.pdf).

Reply:
I will let the documentation of the additional options to another PR as well :)